README.md

```diff
@@ -238,3 +238,29 @@ NOTE: This feature is currently useful when execution is done through the API. T
 Execute the following SQL to upgrade the database:
 
 ```ALTER TABLE `t_st_job_instance` ADD COLUMN `error_message` varchar(4096) CHARACTER SET utf8mb4 COLLATE utf8mb4_bin NULL DEFAULT NULL;```
+
+#### 2. Upgrade from 1.0.2 or before to 1.0.3 or after.
+- Execute the following SQL to upgrade the database:
+  ```
+    ALTER TABLE `user` ADD COLUMN `auth_provider` varchar(10) CHARACTER SET utf8mb4 COLLATE utf8mb4_bin NULL DEFAULT NULL;
```

Comment:
I believe that simply adding a column will not suffice. We also need to insert default value(DB) in the column . Please verify this.